Libraries/PeriphDrivers/Source/SYS/SVD/README.md

```diff
@@ -0,0 +1,10 @@
+SVD Naming Notes:
+
```

Comment:
Do we want to keep this file?  If so, we should add some more text to clarify what these notes mean.